tests/PHPStan/Laravel/Migrations/RequireWithoutTimestampRule/Fixture/database/migrations/invalid_migration.php.inc

```diff
@@ -25,11 +25,9 @@ return new class extends Migration
             'test' => 'test',
         ]);
 
-        $testUser = TestUser::make(['test' => 'test']);
-        $testUser->save();
```

Comment:
PHPStan doesn't seem to like these variables. In migrations, we likely won't be using variables anyway, so I think it should be ok? 👀